app/lib/actions/katello/capsule_content/sync.rb

```diff
@@ -22,22 +22,29 @@ def plan(capsule_content, options = {})
           repository_ids = get_repository_ids(capsule_content, environment, content_view, repository)
           unless repository_ids.blank?
             sequence do
-              plan_action(ConfigureCapsule, capsule_content)
-              plan_action(Pulp::Consumer::SyncNode,
-                          consumer_uuid: capsule_content.consumer_uuid,
-                          repo_ids: repository_ids)
+              plan_action(ConfigureCapsule, capsule_content, environment, content_view)
+
+              smart_proxy = SmartProxy.where(:content_host_id => capsule_content.consumer.id).first
+              fail _("Smart Proxy not found for capsule.") unless smart_proxy
+              concurrence do
+                repository_ids.each do |repo_id|
+                  plan_action(Pulp::Consumer::SyncCapsule,
+                              capsule_id: smart_proxy.id,
+                              repo_pulp_id: repo_id)
+                end
+              end
             end
           end
         end
 
         def get_repository_ids(capsule, environment, content_view, repository)
           if environment
-            repository_ids = capsule.pulp_repos([environment], content_view).map(&:pulp_id)
+            repository_ids = capsule.repos_available_to_capsule(environment, content_view).map(&:pulp_id)
           elsif repository
             repository_ids = [repository.pulp_id]
             environment = repository.environment
           else
-            repository_ids = capsule.pulp_repos.map(&:pulp_id)
+            repository_ids = capsule.repos_available_to_capsule.map(&:pulp_id)
           end
 
           if environment && !capsule.lifecycle_environments.include?(environment)
```

Comment:
Can you add this to this file?  This makes it so capsule syncs do not 'pause' half way through for any error.

```
        def rescue_strategy
            Dynflow::Action::Rescue::Skip
        end
```